encrypted-media-respec.html

```diff
@@ -2547,7 +2557,12 @@
 
         <dt>attribute EventHandler onencrypted</dt>
         <dd>
-          <p>Event handler for the <a def-id="encrypted"></a> event MUST be supported by all HTMLMediaElements as both a content attribute and an IDL attribute.</p>
```

Comment:
Thanks. The simplest thing to do is probably to keep this text^ and duplicate it for the new handler. 

^ We should fix it up to add a break between "event" and "MUST". For example, just add "It MUST... to the new text."